tests/Contrib/Unit/OTLPHttpSpanConverterTest.php

```diff
@@ -0,0 +1,247 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenTelemetry\Tests\Contrib\Unit;
+
+use OpenTelemetry\Contrib\OtlpHttp\SpanConverter;
+use Opentelemetry\Proto\Trace\V1;
+use Opentelemetry\Proto\Trace\V1\InstrumentationLibrarySpans;
+use Opentelemetry\Proto\Trace\V1\ResourceSpans;
+use OpenTelemetry\Sdk\Resource\ResourceInfo;
+use OpenTelemetry\Sdk\Trace\Attribute;
+use OpenTelemetry\Sdk\Trace\Attributes;
+use OpenTelemetry\Sdk\Trace\Clock;
+use OpenTelemetry\Sdk\Trace\Span;
+
+use OpenTelemetry\Sdk\Trace\SpanContext;
+use OpenTelemetry\Sdk\Trace\SpanStatus;
+use OpenTelemetry\Sdk\Trace\TracerProvider;
+
+use PHPUnit\Framework\TestCase;
+
+class OTLPhttpSpanConverterTest extends TestCase
+{
+    /**
+     * @test
+     */
+    public function shouldConvertASpanToAPayloadForOtlp()
+    {
+        $tracer = (new TracerProvider())->getTracer('OpenTelemetry.OtlpTest');
+
+        $timestamp = Clock::get()->timestamp();
+
+        /** @var Span $span */
+        $span = $tracer->startAndActivateSpan('guard.validate');
+        $span->setAttribute('service', 'guard');
+        $span->addEvent('validators.list', $timestamp, new Attributes(['job' => 'stage.updateTime']));
+        $span->end();
+
+        $converter = new SpanConverter();
+        $row = $converter->as_otlp_span($span);
+
+        $this->assertInstanceOf(V1\Span::class, $row);
+
+        $this->assertSame($span->getContext()->getSpanId(), bin2hex($row->getSpanId()));
+        $this->assertSame($span->getContext()->getTraceId(), bin2hex($row->getTraceId()));
+
+        $this->assertSame($span->getSpanName(), $row->getName());
+
+        // $this->assertIsInt($row['timestamp']);
```

Comment:
Are these tests commented out for a reason?

Reply:
Since this file was copied and pasted from OTLP/GRPC, it was from then, which in turn wasn't testing the actual OTLP Format. Likely these need removing. Tests from L92 need looking at, they are valid but don't work which could be related to #357.